WNPRC_Purchasing/resources/views/purchasingLandingPage.html

```diff
@@ -0,0 +1,25 @@
+<div id="<%=h('wnprc_purchasing-report')%>"></div>
+<script type="text/javascript">
+
+    Ext4.onReady(function()
```

Comment:
I don't think we need the onReady() here. Let's avoid ExtJS dependencies when we can.

Reply:
good point, removed